library/common/glfw/embedder.cc

```diff
@@ -300,62 +313,69 @@ bool FlutterInit() {
 
 void FlutterTerminate() { glfwTerminate(); }
 
-PluginRegistrar *GetRegistrarForPlugin(GLFWwindow *flutter_window,
+PluginRegistrar *GetRegistrarForPlugin(FlutterWindowRef flutter_window,
                                        const std::string &plugin_name) {
-  auto *state = GetSavedEmbedderState(flutter_window);
   // Currently, PluginHandler acts as the registrar for all plugins, so the
   // name is ignored. It is part of the API to reduce churn in the future when
   // aligning more closely with the Flutter registrar system.
-  return state->plugin_handler.get();
+  return flutter_window->plugin_handler.get();
 }
 
-GLFWwindow *CreateFlutterWindow(size_t initial_width, size_t initial_height,
-                                const std::string &assets_path,
-                                const std::string &icu_data_path,
-                                const std::vector<std::string> &arguments) {
+FlutterWindowRef CreateFlutterWindow(
```

Comment:
Was that whitespace change deliberately?

Reply:
I'm not sure what whitespace change you mean, but since we use clang-format the answer is almost certainly yes.